spec-0003/index.md

```diff
@@ -17,37 +17,46 @@ If relevant, include examples of how the new functionality would be used,
-Not only should the physical world be accessible, but the digital world as well.
+[Accessibility](https://en.wikipedia.org/wiki/Accessibility) is building things that are fully usable for people with disabilities. [There are many reasons to prioritize accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/#important)), including ethical, user experience, and legal motivations. Accessibility is also a long-recommended web standard (via [WCAG](https://www.w3.org/TR/WCAG/)).
 
-More and more people are accessing the world digitally.
-Access to the internet is a human right.[citation for UN].
-With access to the internet, people can access vital information, such as education [citation for various country laws to access education] or their banking information.
+Scientific Python communities and their projects need to be accessible and inclusive of disabled people. This SPEC is an agreement between projects to 
 
-People with disabilities access the internet as well.
-Blind users can access text on websites with assistive technology such as a screen-reader, which verbally reads text out loud.
-Thus assistive technology empower disabled people to live autonomous lives.
+1. Actively make their work more accessible from software to documentation to community and more. 
+2. Adopt global accessibility guidelines and Scientific Python-specific recommendations as is relevant per project.
+3. Investigate accessibility solutions where they find gaps in existing recommendations (especially for use cases specific to this ecosystem).
+4. Share accessibility knowledge and efforts across projects whenever possible. 
 
-An accessible world allows people with disabilities to not only access information, but also communicate and create.
-Helen Keller’s access to Braille, another form of assistive technology, allowed her to read about the world.
-She used Braille to communicate with others who were Deaf-Blind.
-It also allowed her to publish her experiences [citation for Story of My Life] and advocate for the Deaf-Blind community to the general public.
+## Community
 
-Accessibility benefits everyone.
-For example, captions for videos benefit not only Deaf viewers with complete hearing loss, but also viewers with partial hearing loss due to age.
-Captions also benefit viewers who cannot play audio out loud, such as in a quiet environment or if they have broken speakers.
-[citation to range of disabilities and situational disability].
-Thus accessibility benefits not just “a few disabled people”[a] but all people.
+At the time of writing, accessibility is not a regular consideration in many Scientific Python projects. Many projects may have a handful of people who have accessibility knowledge or are interested in accquiring more, but may not be able to reach critical mass on a per-project basis. Connecting Scientific Python community members focused on accessibility will provide a boon for individuals, projects, and the ecosystem at large. 
```

Comment:
Later once the url is live, we can link to accessibility.scientific-python.org/community page, which is the community.rst page